nixos/modules/services/x11/desktop-managers/gnome.nix

```diff
@@ -547,7 +547,7 @@ in
 
       # Let nautilus find extensions
       # TODO: Create nautilus-with-extensions package
-      environment.sessionVariables.NAUTILUS_EXTENSION_DIR = "${config.system.path}/lib/nautilus/extensions-3.0";
+      environment.sessionVariables.NAUTILUS_EXTENSION_DIR = "${config.system.path}/lib/nautilus/extensions-4";
```

Comment:
Could we get the version number via nix or is that not easily possible?

Reply:
Not without IFD or storing it in passthru.